media-gfx/sane-frontends/sane-frontends-1.0.14-r4.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Scanner Access Now Easy"
+HOMEPAGE="http://www.sane-project.org"
+SRC_URI="https://alioth.debian.org/frs/download.php/file/1140/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="gimp gtk"
+
+RDEPEND="
+	media-gfx/sane-backends
+	gimp? ( media-gfx/gimp:2 )
+	gtk? (
+		dev-libs/glib:2
+		x11-libs/gtk+:2
+	)
+"
+DEPEND="${RDEPEND}"
+
+REQUIRED_USE="gimp? ( gtk )"
+
+DOCS=( AUTHORS Changelog NEWS PROBLEMS README )
+
+PATCHES=( "${FILESDIR}/MissingCapsFlag.patch" )
+
+src_configure() {
+	econf \
+		--datadir=/usr/share/misc \
+		$(use_enable gimp) \
+		$(use_enable gtk gtk2) \
+		$(use_enable gtk guis)
+}
+
+src_install() {
+	local gimpplugindir
+	local gimptool
+	emake DESTDIR="${D}" install
+
+	# link xscanimage so it is seen as a plugin in gimp
+	if use gimp; then
+		local plugindir
+		if [ -x "${EPREFIX}"/usr/bin/gimptool ]; then
```

Comment:
Use `[[ ... ]]` in bash, please. This also renders quoting inside unnecessary.

Also, why are you combining full path here with basename call below? Maybe you want to check for `type gimptool &>/dev/null` instead?